Dockerfile.localdev

```diff
@@ -0,0 +1,43 @@
+##
+# This Dockerfile is used to build the uptime-checker binary for local development.
+# It is not used for production. We need this because certain error messages
+# are Linux specific so we have to run the binary on Linux to get the correct
+# error messages.
+
+# HOW TO USE:
+# 1. Build the binary: `docker build -f Dockerfile.localdev -t my-uptime-checker`
+# 2. Run the container with a mounted volume for src: `docker run -it -v $(pwd)/src:/app/src --entrypoint /bin/sh my-uptime-checker`
+# e. in the container you can do `cargo test`, etc. and updating the src directory on your host will reflect in the container,
+# so you can test your changes without having to rebuild the container.
+##
```

Comment:
Thanks for this, helpful to both have this container and easy instructions on how to use it.